webdriver-spec.html

```diff
@@ -1986,8 +1986,9 @@ <h2 data-dfn-for="NavigatorAutomationInformation">Interface</h2>
  <tr>
   <td><dfn><code>noProxy</code></dfn>
   <td>array
-  <td>Lists the address for which the proxy should be bypassed when
-   the <a><code>proxyType</code></a> is "<code>manual</code>".
+  <td>Comma-separated list of hosts for which the proxy should be bypassed if
```

Comment:
The `Valid Values` column tells us that this is not a comma-separated list, but a JSON `List` type:

> A List containing any number of any of domains, IPv4 addresses, or IPv6 addresses.

If the driving force behind this is to make the implementation less fussy (for example, just a `String` that the implementation can do what it wants with), then you also need to modify the `Valid Value` column.

Reply:
Oh, so it means we are requiring something like `{"noProxy": ["foo.bar", "127.0.0.1"]}`?

With that the different driver implementations would have to join all entries into a single string with a separator the browser supports. I would be fine with that, and would update the PR to only make this entry sound similar to `proxyAutoconfigUrl`.

Reply:
@shs96c any further feedback? If the last comment applies to how it should look like, we can close this issue. Or should we mention "JSON list" similar to other parts of the spec?

Reply:
Yes, the spec is saying that we should have something like `{"noProxy": ["foo.bar"]}`

Reply:
So should we update to say "JSON list"? I can update my PR appropriately.